set.mm

```diff
@@ -346528,58 +346648,6 @@ or are almost disjoint (the interiors are disjoint).  (Contributed by
-      syl3anc nnne0d cmul itgmulc2 eqidd oveq1 oveq2d adantl simpr ioossicc a1i
-      cioo syldan mulcld fvmptd itgeq2dv cdv crn ctg ccnfld cpr reelprrecn 1nn0
-      ctopn nn0addcld nn0cnd 1cnd addcld cdm fmpttd ssidd dvexp pncand mpteq2dv
-      wf eqtrd feq1d mpbird fdmd sseqtrrid dvres3 syl22anc reseq1d mp1i 3eqtr3d
-      resmpt tgioo2 cnt iccntr dvmptres2 ioossre sstri cncfmptc mulcncf eqeltrd
-      eqid cvol ioombl iblss ftc2 wral fveq1d ralrimivw itgeq2 oveq1d cxr rexrd
-      cle wbr ubicc2 recnd lbicc2 oveq12d itgioo 3eqtr3rd mvllmuld ) AEUAUBKZBC
-      DUCKZBUDZELKZUEZDUUELKZCUUELKZUFKZAUUEAEUGMZUUEUHMZIEUIUJZUKZABUUFUUHNAUU
-      GUUFMZURZUUGEAUUFNUUGAUUFONACOMZDOMZUUFOULFGCDUMUNZUOUPZUQAUUMUUQIPQZAUUS
-      UUTBUUFUUHRZUUFNUSKZMUVDUTMFGABNUUHRZUUFVAZUVDUVEABNUUFUUHUVBVBAUUFNULZUV
-      FNNUSKZMZUVGUVEMUVBAUUMUVJIBEVCUJNNUUFUVFVDVEVFCDUVDVGVIZVHAUUEUUOVJAUUEU
-      UIVKKBUUFUUEUUHVKKZUEZUULABUUFUUHUUENUUPUVCUVKVLABCDVTKZUUGJUVNUUEJUDZELK
-      ZVKKZRZSZUEZBUVNUVLUEUULUVMABUVNUVSUVLAUUGUVNMZURZJUUGUVQUVLUVNUVRNUWBUVR
-      VMUVOUUGTZUVQUVLTUWBUWCUVPUUHUUEVKUVOUUGELVNVOVPAUWAVQUWBUUEUUHAUUENMZUWA
-      UUPPAUWAUUQUUHNMAUVNUUFUUGUVNUUFULACDVRVSZUQUVCWAWBWCWDABUVNUUGOJUUFUVOUU
-      ELKZRZWEKZSZUEZDUWGSZCUWGSZUFKUVTUULABCDUWGFGHAUWHUVRUVNNUSKZAJUWFUVQOVTW
-      FWGSZWHWLSZNOUVNUUFOONWIMZAWJVSZAUVOOMZURZUVOUUEAONUVOONULZAUOVSUQZAUUEUG
-      MZUWRAEUAIUAUGMAWKVSWMZPQUWSUUEUVPUWSEUAAENMZUWRAEIWNZPUWSWOWPUWSUVOEUXAA
-      UUMUWRIPQWBAOJNUWFRZOVAZWEKZJNUVQRZOVAZOJOUWFRZWEKJOUVQRZAUXHNUXFWEKZOVAZ
-      UXJAUWPNNUXFXCNNULZOUXMWQZULUXHUXNTUWQAJNUWFNAUVONMZURZUVOUUEAUXQVQZAUXBU
-      XQUXCPQWRANWSZANOUXPUOANNUXMANNUXMXCNNUXIXCAJNUVQNUXRUUEUVPAUWDUXQUUPPUXR
-      UVOEUXSAUUMUXQIPQWBWRANNUXMUXIAUXMJNUUEUVOUUEUAUFKZLKZVKKZRZUXIAUUNUXMUYD
-      TUUOJUUEWTUJAJNUYCUVQAUYBUVPUUEVKAUYAEUVOLAEUAUXEAWOXAVOVOXBXDZXEXFXGXHNO
-      UXFXIXJAUXMUXIOUYEXKXDAUXGUXKOWEUWTUXGUXKTAUOJNOUWFXNXLVOUWTUXJUXLTAUOJNO
-      UVQXNXLXMUVAUWOUWOYDZXOUYFAUUSUUTUUFUWNXPSSUVNTFGCDXQUNXRZAJUUEUVPUVNAUWD
-      UVNNULZUXOJUVNUUERUWMMUUPUYHAUVNONCDXSUOXTZVSZUXTJUUEUVNNYAVIAJNUVPRZUVNV
-      AZJUVNUVPRZUWMUYHUYLUYMTAUYIJNUVNUVPXNXLAUYHUYKUVIMZUYLUWMMUYJAUUMUYNIJEV
-      CUJZNNUVNUYKVDVEVFYBYCAUWHUVRUTUYGAJUVNUUFUVQNUWEUVNYEWQMACDYFVSAUVOUUFMZ
-      URZUUEUVPUYQEUAAUXDUYPUXEPUYQWOWPUYQUVOEAUUFNUVOUVBUQAUUMUYPIPQWBAUUSUUTJ
-      UUFUVQRZUVEMUYRUTMFGAJUUEUVPUUFAUWDUVHUXOJUUFUUERUVEMUUPUVBUXTJUUEUUFNYAV
-      IAUYKUUFVAZJUUFUVPRUVEAJNUUFUVPUVBVBAUVHUYNUYSUVEMUVBUYONNUUFUYKVDVEVFYBC
-      DUYRVGVIYGYCAUXFUUFVAZUWGUVEAJNUUFUWFUVBVBAUVHUXFUVIMZUYTUVEMUVBAUXBVUAUX
-      CJUUEVCUJNNUUFUXFVDVEVFYHAUWIUVSTZBUVNYIUWJUVTTAVUBBUVNAUUGUWHUVRUYGYJYKB
-      UVNUWIUVSYLUJAUWKUUJUWLUUKUFAJDUWFUUJUUFUWGNAUWGVMZAUVODTZURUVODUUELAVUDV
-      QYMACYNMZDYNMZCDYPYQZDUUFMACFYOZADGYOZHCDYRVIADUUEADGYSUXCQWCAJCUWFUUKUUF
-      UWGNVUCAUVOCTZURUVOCUUELAVUJVQYMAVUEVUFVUGCUUFMVUHVUIHCDYTVIACUUEACFYSUXC
-      QWCUUAXMABCDUVLFGUURUUEUUHAUWDUUQUUPPUVCWBUUBUUCXDUUD $.
-  $}
 
```

Comment:
one of the two blank lines should be removed.

Reply:
I hope I've caught them all